modules/ROOT/pages/create-an-api-configure-service-mesh-CRD.adoc

```diff
@@ -11,11 +11,19 @@ After you provision the adapter, you must create an API to bind to the service.
 If you want to create REST or SOAP APIs with OAS or WSDL definition, use Anypoint Platform.
 +
 Note that if you want to upload a REST API or OAS asset type, use a .json or .yaml file. Alternatively, if you want to upload a SOAP API or WSDL asset type, use a .wsdl or .xml file.
-* <<create_api-CRD, Create an API using Anypoint Service Mesh Autocreate>>
+* <<create_api-CRD, Create an API using Anypoint Service Mesh Autocreate with user credentials>>
 +
-Anypoint Service Mesh enables you to automatically create an HTTP API in Anypoint Platform, with no REST or SOAP definitions. You do not have to manually log in to Anypoint Platform. 
+Anypoint Service Mesh enables you to automatically create an HTTP API in Anypoint Platform using username and password, with no REST or SOAP definitions. You do not have to manually log in to Anypoint Platform. 
 +
 If you automatically create REST or SOAP APIs with Anypoint Service Mesh, the APIs are still managed internally as an HTTP asset without any definition.
+* <<create-api-CRD-ConnAps, Create an API using Anypoint Service Mesh Autocreate with client credentials>>
++
+Anypoint Service Mesh enables you to automatically create an HTTP API in Anypoint Platform using the credentials you create with xref:access-management::connected-apps-overview.adoc[Connected Apps]. You do not have to manually log in to Anypoint Platform.
```

Comment:
What do you think of changing the wording for something like:
```suggestion
Anypoint Service Mesh enables you to automatically create an HTTP API in Anypoint Platform using the credentials of xref:access-management::connected-apps-overview.adoc[Connected Apps]. You do not have to manually log in to Anypoint Platform.
```
Please, if you choose to apply my suggestion, apply it also in `create-an-api-configure-service-mesh-CLI.adoc`

Reply:
Done! Thanks!